cmd/kube-controller-manager/app/options/options.go

```diff
@@ -196,8 +197,8 @@ func NewKubeControllerManagerOptions() (*KubeControllerManagerOptions, error) {
 	}
 
 	s.GarbageCollectorController.GCIgnoredResources = gcIgnoredResources
-	s.Generic.LeaderElection.ResourceName = "kube-controller-manager"
-	s.Generic.LeaderElection.ResourceNamespace = "kube-system"
+	s.Generic.LeaderElection.ResourceName = KubeControllerManagerUserAgent
```

Comment:
I would revert this one, it's not clear the user agent controls functional behavior like the lock resource name or key pair filename

Reply:
i agree